main.go

```diff
@@ -99,6 +99,7 @@ var (
 	syncPeriod                    time.Duration
 	restConfigQPS                 float32
 	restConfigBurst               int
+	drainKubeTimeout              time.Duration
```

Comment:
```suggestion
	nodeDrainClientTimeout              time.Duration
```
nit: I would go with this. Simlar to NodeDrainTimeout but it's clear that it's for the client.

(I think we don't need kube as everything around here is Kubernetes ;) )

Reply:
+1. May be worth commenting how they relate. e.g. Setting the client timeout higher than spec.NodeDrainTimeout will effectively increase the latter.